src/Catty/Helpers/BrickHelpers/BubbleBrickHelper.h

```diff
@@ -1,36 +0,0 @@
-/**
```

Comment:
This deprecated file is not used anymore (converted to Swift) and has been removed from the file system.